inference-engine/src/low_precision_transformations/src/fuse_multiply_to_fake_quantize.cpp

```diff
@@ -32,26 +32,26 @@ bool FuseMultiplyToFakeQuantizeTransformation::transform(TransformationContext&
 
     const auto multiplyConstant = multiply->get_input_node_shared_ptr(1);
 
-    auto outputLowConst = fakeQuantize->get_input_node_shared_ptr(3);
-    auto outputHighConst = fakeQuantize->get_input_node_shared_ptr(4);
+    auto outputLowConst_f32 = fold<opset1::Convert>(fakeQuantize->get_input_node_shared_ptr(3), deqPrecision);
```

Comment:
minor: 2021 is today, update header: `// Copyright (C) 2020-2021 Intel Corporation` and in OTHER files